lesson_15/tests/test_polyon.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from lesson_15.lesson_15_mihaela_vaida import polygon
+
+
+class TestPolygon(unittest.TestCase):
+    def test_polygon_object_len_of_sides(self):
+        sides = 4, 3, 4, 5
+        test_polygon = polygon.Polygons(sides)
+        self.assertGreaterEqual(len(sides),3, "Not a valid polygon!")
+
+    def test_polygon_object_sides_type(self):
+
+       test_polygon = polygon.Polygons(3, 4, 5)
+       self.assertIsInstance(3, int)
+       self.assertIsInstance(4, int)
+       self.assertIsInstance(5, int)
+
+    def test_polygon_perimeter_sum(self):
+        a,b,c=3,5,4
+        test_polygon = polygon.Polygons(3, 5, 4)
+        perimeter=a+b+c
```

Comment:
same as above, make sure you youse the code. here perimeter is test_polygon.perimeter()

Reply:
modified !